tendrl/commons/flows/expand_cluster/__init__.py

```diff
@@ -0,0 +1,131 @@
+# flake8: noqa
```

Comment:
We have to fix PEP8 if the checks fail

Also NS will be considered as a builtin, are you sure you are running the correct command for pep8? "tox -epep8" ?

https://github.com/Tendrl/commons/blob/develop/tox.ini#L74